StudCity.Frontend/src/components/ScrollerPagination/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useRef, useEffect, useState } from 'react';
+import Spinner from '../Spinner';
+
+const ScrollerPagination = ({
+	loading,
+	data,
+	ItemComponent,
+	handleClick,
+	handleDispatch,
+	searchData,
+	selectedItems = []
+}) => {
+	const containerRef = useRef(null);
+	const [indexPage, setIndexPage] = useState(1);
+	const [datas, setDatas] = useState([]);
```

Comment:
word 'data' is already means multiple items, you don't need an 's' at the end

Reply:
@Serhiy-dotcom @Vova-Romaniuk I think items will be better